examples/threadExample/src/main/java/io/realm/examples/threads/ThreadExampleActivity.java

```diff
@@ -84,10 +82,13 @@ public SectionsPagerAdapter(FragmentManager fm) {
 
         @Override
         public Fragment getItem(int position) {
-            switch(position) {
-                case 0: return new ThreadFragment();
```

Comment:
Yes I would allow that as well. Bigger switches can actually be much more readable if allowed. Even easier to see if break is missing.